ext/standard/random.c

```diff
@@ -158,19 +159,19 @@ PHP_FUNCTION(random_int)
 	zend_ulong umax;
 	zend_ulong result;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ll", &min, &max) == FAILURE) {
+	if (zend_parse_parameters_throw(ZEND_NUM_ARGS(), "ll", &min, &max) == FAILURE) {
 		return;
 	}
 
 	if (min >= max) {
```

Comment:
@bwoebi [This PR](https://github.com/SammyK/php-src/blob/csprng-error-conditions/ext/standard/random.c#L166-L173) adjusted the logic to accept `random_int(42, 42)`.

@ircmaxell Since the RFC passed but with the `min > max` spec, does this not get adjusted?